apps/protocol-frontend/src/components/ReportForm.tsx

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
+// TODO: JP, would you disable this ^ and check errors.
```

Comment:
@jonathanprozzi I can use your help here. Would you disable this flag and check errors? 

They are interesting ones.